OmiseSwift/Source.swift

```diff
@@ -330,8 +330,27 @@ public enum Wallet: Codable, Equatable {
     
     public struct AlipayWallet: Codable, Equatable {
         public let barcode: String
-        public let storeID: String
-        public let storeName: String
+        
+        public struct StoreInformation: Codable, Equatable {
+            public let storeID: String
+            public let storeName: String
+            
+            public init(storeID: String, storeName: String) {
```

Comment:
So we must have both or none at all, right?

Reply:
Correct